### PR TITLE
new homepage for mina.granola.team

### PR DESCRIPTION
**Pull request by jstavh**
This PR is a nearly complete re-do of the mina.granola.team website. One card points to Mo's recent project (the recent github repo). The second card currently points to home given no markdown files, but we can point it to markdown files as initial perspective. As requested, there is a nav bar. The projects button in the nav bar points to Mo's recent github repo for now. Footer is always at the bottom. Site is responsive. Hover effect on cards is "ease" transition but can make it fast if you want it that way for this site as well. Last but not least, this code also builds.

### CODE REVIEW

**Comment by robinbb** [comment]
@jstavh Instead of the Granola Blog Posts and articles link, can you make it a link to the on-chain signaling project?

**Comment by jstavh** [comment]
@robinbb I made this change in the recent commit. For the nav bar Projects button, what would you like it to point to?

File: src/astro/src/components/Front.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            <div className="flex flex-col mt-6 px-12 justify-center">
+                <div className="flex flex-col items-center">
+                    <a href="https://github.com/Granola-Team/mina-security-report" 
+                    className="card text-3xl text-white hover:text-logoOrange
+                    focus:text-logoOrange active:text-logoOrange">
+                        Mina Security Report made by Granola &rarr; 
+                        <br></br><br></br>
+                        <span className="text-2xl">Come see our source code.</span>
+                    </a>
+                    <a href="/" className="card text-3xl text-white hover:text-logoOrange
```
Comment (robinbb) [inline src/astro/src/components/Front.jsx]: This still does not lead anywhere because href="/'

**Review by robinbb — CHANGES_REQUESTED** [review]

File: src/astro/src/components/Front.jsx (hunk 1 above)
Comment (jstavh) [reply src/astro/src/components/Front.jsx]: Question: Would you like this to point to https://github.com/Granola-Team/onchain-signalling or https://github.com/Granola-Team/mina-signaling

**Comment by jstavh** [comment]
OCS points to slug onchain-signalling vs mina-signaling and removed Projects tab from nav-bar. Netlify preview shows up to date display.

**Pull request merged**